File: app/src/main/java/com/example/getfood/ui/loginregister/RegisterMvpView.java
package com.example.getfood.ui.loginregister;

import android.content.Context;

import com.example.getfood.ui.base.BaseView;

public interface RegisterMvpView extends BaseView {

    Context getContext();

    void valueEntryError(String errorMsg);

    void onUserCreatedSuccessfully();

    void userVerifiedSuccessfully();

    void onUserEmailVerificationFailed();
}

File: app/src/main/java/com/example/getfood/ui/loginregister/registerfragment/RegisterMvpView.kt
package com.example.getfood.ui.loginregister.registerfragment

import com.example.getfood.ui.base.BaseView

interface RegisterMvpView : BaseView {

    fun valueEntryError(errorMsg: String?)
    fun onUserCreatedSuccessfully()
    fun userVerifiedSuccessfully()
    fun onUserEmailVerificationFailed()
}